core/consensus/babe/src/lib.rs

```diff
@@ -15,22 +15,66 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 //! BABE (Blind Assignment for Blockchain Extension) consensus in substrate.
-#![forbid(warnings, unsafe_code, missing_docs)]
-
+//!
+//! Note that 1 second slot time is optimal.
```

Comment:
is it?

Reply:
a doc about instablity would be nice

Reply:
We should omit this statement from the docs until any other parameters are documented as well.